templates/kubernetes/docs/base_docs.html

```diff
@@ -4,6 +4,10 @@
 
 {% block meta_copydoc %}https://drive.google.com/drive/folders/1FyMcu6vqGnyQMdWLRjvxJVWm8gHgslk4{% endblock meta_copydoc %}
 
+{% block head_extra %}
+<link rel="stylesheet" href="/static/css/prism.css" /><script src="/static/js/prism.js"></script>
```

Comment:
Can you put these two imports on different lines please. Easy to see whats going on